src/wallet.rs

```diff
@@ -757,10 +768,10 @@ mod tests {
         let wallet =
             Wallet::new_from_seed("wallet-1".to_string(), &[1u8; 32], Chain::Elements).unwrap();
 
-        let encryption_key = [0u8; 32];
+        let password = [0u8; 32];
```

Comment:
Can we use a more password-y value here? We take everything that is `AsRef<[u8]>`, meaning we can actually use a string here!